CMakeLists.txt

```diff
@@ -217,6 +218,7 @@ message(STATUS "Ubuntu 14: ${SSG_PRODUCT_UBUNTU14}")
 message(STATUS "Ubuntu 16: ${SSG_PRODUCT_UBUNTU16}")
 message(STATUS "WRLinux: ${SSG_PRODUCT_WRLINUX}")
 message(STATUS "Oracle Linux 7: ${SSG_PRODUCT_OL7}")
+message(STATUS "Example: ${SSG_PRODUCT_EXAMPLE}")
```

Comment:
This cosmetic change, but it can be more visible if on top of the list.

Reply:
I've moved everything about this product to be at the same position (top) of their respective sections.